CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * [FEATURE] Compactor: Added `-compactor.block-files-concurrency` allowing to configure number of go routines for download/upload block files during compaction. #4784
 * [ENHANCEMENT] Querier/Ruler: Retry store-gateway in case of unexpected failure, instead of failing the query. #4532
 * [FEATURE] Compactor: Added -compactor.blocks-fetch-concurrency` allowing to configure number of go routines for blocks during compaction. #4787
+* [ENHANCEMENT] Querier: Report `fetched_series_count` stats for GetSeries calls from ingesters. #4797
```

Comment:
Few things

* This looks more like a bug fix more than enhancement
* The component modified seems to be distributor not querier (although the call path is from querier)
* the `GetSeries` calls are **to** ingesters, not "from" ingesters right?

Reply:
Addressed these comments. :)